clang/test/CodeGen/rtsan_attribute_inserted.c

```diff
@@ -1,7 +1,11 @@
 // RUN: %clang_cc1  -triple x86_64-unknown-linux -fsanitize=realtime %s -emit-llvm -o - %s | FileCheck %s
 
 float process(float *a) [[clang::nonblocking]] { return *a; }
-
-// CHECK-LABEL: @process{{.*}}#0 {
+// CHECK: @process{{.*}} #0 {
```

Comment:
Don't we still want check-label here? If i understand correctly what that does, it breaks the checks into sections, which make them more strict (you can't match things out of order)

Reply:
Unfortunately the test fails if we keep `CHECK-LABEL` for both `process` and `spinlock`. This is because the attributes `#0` and `#1` are defined at the bottom of the file, so the ordering of the checks is broken. I think this removal of `CHECK-LABEL` is OK - the attributes are matched to the functions by their `#{number}`, so the test is still checking the right thing :)